routes/web.php

```diff
@@ -36,6 +36,12 @@
     ->get('/results', [ResultsController::class, 'index'])
     ->name('results');
 
+Route::middleware('simulateError')
+    ->get('/mismatch-review', function () {
```

Comment:
I don't think this is going to be a get request, also I don't think this is needed here at the moment. What I thought we would do is add an `update` method to the controller, and as such, we can also apply this middleware at the controller level.

Reply:
moved into the ResultsController and changed to PUT. While it's not needed for functionality right now, the test with its simulated error still relies on this endpoint.